app/ui/shared/widgets/autocompleted-search.jsx

```diff
@@ -35,13 +35,30 @@ class AutocompletedSearch extends Component {
   constructor(props) {
     super(props);
     this.shouldComponentUpdate = PureRenderMixin.shouldComponentUpdate.bind(this);
+    this.handleWindowClick = this.handleWindowClick.bind(this);
 
     this.state = {
       showSelectionList: false,
       selectedIndex: 0,
     };
   }
 
+  componentWillMount() {
+    if (typeof window === 'object') {
+      window.addEventListener('click', this.handleWindowClick);
```

Comment:
Why the need to check if the window global exists? I thought you said this was an eslint error.